contracts/AgentMech.sol

```diff
@@ -401,6 +478,7 @@ contract AgentMech is ERC721Mech {
         _locked = 1;
     }
 
+    /// TODO: We need the same system to work with Nevermined pricing.
```

Comment:
Important that we consider this right from the start. Ideally the factory takes care of creating both types of mechs